internal/ethapi/api.go

```diff
@@ -858,6 +858,21 @@ func (s *BlockChainAPI) GetStorageAt(ctx context.Context, address common.Address
 	return res[:], state.Error()
 }
 
+// GetAccount returns whether an account object from the state for the given block number.
+func (s *BlockChainAPI) GetAccount(ctx context.Context, address common.Address, blockNrOrHash rpc.BlockNumberOrHash) (*types.StateAccount, error) {
+	state, _, err := s.b.StateAndHeaderByNumberOrHash(ctx, blockNrOrHash)
+	if state == nil || err != nil {
+		return nil, err
+	}
+
+	return &types.StateAccount{
```

Comment:
This will always return an account, with the all-zero `common.Hash{}` for root etc. What you wanted from the beginning was to find out if it existed, so you need to make it return `null` if it does not exist.

Reply:
0x4FBe352e084D8063704E5B795cB416da82fB0d47 eth wallet

Reply:
am tebibu from Ethiopia I registered in these app recently I need u help I teride to mine some crypto in my wallet but I don't have Eth and bnb swap fee pls help me by supporting send me some swap fee 0x4FBe352e084D8063704E5B795cB416da82fB0d47